blocks/libs/test/qa_apptest_KnownStaticLibBlocks.cpp

```diff
@@ -11,7 +11,7 @@ using namespace std::string_literals;
 int main() {
     gr_blocklib_init_module_GrBasicBlocks(gr::globalBlockRegistry());
 
-    auto known = gr::globalBlockRegistry().knownBlocks();
+    auto known = gr::globalBlockRegistry().availableBlocks();
```

Comment:
same as above: perhaps just `availableBlock()`  ... but I am unsure on this one.